src/test/java/de/tum/cit/aet/artemis/lecture/ExerciseUnitIntegrationTest.java

```diff
@@ -144,16 +144,16 @@ void createExerciseUnit_asStudent_shouldForbidden() throws Exception {
         ExerciseUnit exerciseUnit = new ExerciseUnit();
         exerciseUnit.setExercise(exercise);
         exerciseUnit.setId(1L);
-        request.postWithResponseBody("/api/lecture/lectures/" + lecture1.getId() + "/exercise-units", exerciseUnit, ExerciseUnit.class, HttpStatus.BAD_REQUEST);
+        request.postWithResponseBody("/api/lecture/lectures/" + lecture1.getId() + "/exercise-units", exerciseUnit, ExerciseUnit.class, HttpStatus.FORBIDDEN);
     }
```

Comment:
I think there is a mistake in this test, shouldn't the role be `STUDENT`?

```suggestion
@WithMockUser(username = TEST_PREFIX + "student1", roles = "STUDENT")
void createExerciseUnit_asStudent_shouldForbidden() throws Exception {
Exercise exercise = course1.getExercises().stream().findFirst().orElseThrow();
ExerciseUnit exerciseUnit = new ExerciseUnit();
exerciseUnit.setExercise(exercise);
exerciseUnit.setId(1L);
        request.postWithResponseBody("/api/lecture/lectures/" + lecture1.getId() + "/exercise-units", exerciseUnit, ExerciseUnit.class, HttpStatus.FORBIDDEN);
}
```